README.md

```diff
@@ -83,6 +83,7 @@ You can check a list of all the environment variables [here](vars.md)
 ./01_prepare_host.sh
 ./02_configure_host.sh
 ./03_launch_mgmt_cluster.sh
+./04_verify.sh
```

Comment:
`make` target runs all 4 scripts, so this makes sense to me

Reply:
Yes, that's exactly why I added it.